### PR TITLE
Html layout and Book Storage

### PR DESCRIPTION
**Pull request by Bavon101**
At this stage of the project, @levy002  and I were able to archive the following :

- [ ] Created a collection that keeps a list of books, an `array` of book objects that is stored locally.
- [ ] Created a function to add a new book to the collection, with title and author, which is uniquely identified by the `id`.
- [ ] Created a function to remove a book from the collection using the book `id`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@Bavon101](https://github.com/Bavon101)
+<!-- - Twitter: [@twitterhandle](https://twitter.com/twitterhandle) -->
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/akumu-bavon-335416193/)
+
```
Comment (mikethreels) [inline README.md]: - [x] please include all authors of this project in the readme file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Acknowledgments
+
+- Microverse offered guidance on how to make the project more professional
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (mikethreels) [inline README.md:76]: - [x] the link to the license is not working as expected, please make sure it works as shown in [the template](https://github.com/microverseinc/readme-template) provided by Microverse.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+### Deployment -->
+
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@Bavon101](https://github.com/Bavon101)
+- GitHub: [@levy002](https://github.com/levy002)
```
Comment (mateo951) [inline README.md:54]: Addressed!

File: README.md (hunk 2 above)
Comment (mateo951) [reply README.md:76]: Addressed!

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done making all the necessary changes requested by the previous reviewer!
Your project is complete!  There is nothing else to say other than... it's time to merge it 👏 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**